models/team_test.go

```diff
@@ -58,6 +59,68 @@ func TestCreateTeam(t *testing.T) {
 	}
 }
 
+// TestDestroyTeam test that you can destroy a team.
+//
+func TestDestroyTeam(t *testing.T) {
+	var c aetest.Context
+	var err error
+	options := aetest.Options{StronglyConsistentDatastore: true}
+
+	if c, err = aetest.NewContext(&options); err != nil {
+		t.Fatal(err)
+	}
+	defer c.Close()
+
+	tests := []struct {
+		title      string
+		team       testTeam
+		overrideId bool
+		newId      int64
+		err        string
+	}{
+		{
+			title: "can destroy team",
+			team:  testTeam{"my team", "description", 10, false},
+		},
+		{
+			title:      "cannot destroy team",
+			team:       testTeam{"my team", "description", 10, false},
+			overrideId: true,
+			newId:      11,
+			err:        "Cannot find team with Id",
+		},
+	}
+
+	for i, test := range tests {
+		t.Log(test.title)
+		var got *Team
+		if got, err = CreateTeam(c, test.team.name, test.team.description, test.team.adminId, test.team.private); err != nil {
```

Comment:
Maybe it is better to do first the creation of the team then define the testing structure then test first "cannot find" test case then the "can destroy". What do you think ?